swig/python/osgeo/gdal.py

```diff
@@ -962,25 +962,42 @@ def VectorTranslateOptions(options=None, format=None,
          forceNullable=False,
          emptyStrAsNull=False,
          spatFilter=None, spatSRS=None,
+         srcDatasetOpenOptions=None,
+         dstDatasetOpenOptions=None,
          datasetCreationOptions=None,
          layerCreationOptions=None,
          layers=None,
          layerName=None,
          geometryType=None,
          dim=None,
-         segmentizeMaxDist= None,
+         transactionSize=None,
+         clipSrc=None,
+         clipSrcSQL=None,
+         clipSrcLayer=None,
+         clipSrcWhere=None,
+         clipDst=None,
+         clipDstSQL=None,
+         clipDstLayer=None,
+         clipDstWhere=None,
+         simplifyTolerance=None,
+         segmentizeMaxDist=None,
          makeValid=False,
+         mapFieldType=None,
+         explodecollections=False,
```

Comment:
camelCase to apply here and in other related places:
```suggestion
         explodeCollections=False,
```